tests/integration/test_schema_migration.py

```diff
@@ -334,6 +336,7 @@ def test_migrate_12(
         assert row.urls_metadata is None
 
 
+@pytest.mark.skip(reason="Test does not execute inner function")
```

Comment:
What is the plan for the skipped tests? Is there a plan to update them? Are they still needed? Should they be removed?

Reply:
these should be removed when we clean up the deprecated models